datalab/datalab_session/analysis/raw_data.py

```diff
@@ -45,9 +43,20 @@ def raw_data(input: dict):
             if not max_value:
                 max_value = np.finfo(datatype).max
 
-    scaled_array = cv2.resize(image_data, dsize=(max_size, max_size), interpolation=cv2.INTER_AREA)
+    # convert to 64 bit for resizing to avoid NaN values
+    image_data = sci_hdu.data
+    image_data_64 = image_data.astype(np.float64)
+
+    # Resize, convert back to original datatype and flip the image
+    scaled_array_64 = cv2.resize(image_data_64, dsize=(max_size, max_size), interpolation=cv2.INTER_AREA)
+    scaled_array = scaled_array_64.astype(datatype)
     scaled_array_flipped = np.flip(scaled_array, axis=0)
 
+    # Compute the fits2image autoscale params to send with the image
+    samples = extract_samples_in_place(image_data_64, sci_hdu.header.get('NAXIS1'), sci_hdu.header.get('NAXIS2'))
```

Comment:
This needs to use the original data, not the scaled down data since it itself scales down the data to get samples...